### DISCUSSION TITLE
Unkillable alerts

### DISCUSSION THREAD

**Discussion by Sagatjkeee** [Security-Onion-Solutions/securityonion#8077] [Unsupported Versions]
Hi!
I create and deleted Playbook, he have status code 404 in /playbook/issues/*, and not have in /opt/so/rules/elastalert/playbook/*yaml
why this alert is continues to be generated?
Same name is not in playbook and rule.uuid (PlayID) too.
I will be very grateful if you give me a prompt!
Version SOC: 2.3.120

**Comment by dougburks** [comment]
Have you tried restarting Playbook?
```
sudo so-playbook-restart
```

**Reply by Sagatjkeee** [reply]
did not help ._.

**Reply by defensivedepth** [reply]
@Sagatjkeee Please confirm that the rule is not in `/opt/so/rules/elastalert/playbook/`, then restart elastalert:

`so-elastalert-restart --force`

**Reply by Sagatjkeee** [reply]
this command will reset my rules where i edited the logic elastalert?
if yea, i will be wait update to .130 :> 
this file  in /opt/so/rules/elastalert/playbook/ is - "No such file or directory"

**Reply by defensivedepth** [reply]
@Sagatjkeee Are you saying that you manually edited files in /opt/so/rules/elastalert/playbook/ ?

**Reply by btdewey** [reply]
Did you run that ls with sudo?  Need root permissions to see the directory.

**Reply by btdewey** [reply]
If you get a file that can't properly be converted it will cause the rule update process to break and then rules don't get updated.
